com.unity.perception/Runtime/Randomization/Scenarios/ScenarioBase.cs

```diff
@@ -324,28 +335,13 @@ public void ReorderRandomizer(int currentIndex, int nextIndex)
         }
 
         /// <summary>
-        /// Generates a random seed by hashing the current scenario iteration with a given base random seed
+        /// Generates and returns a new random seed
         /// </summary>
-        /// <param name="baseSeed">Used to offset the seed generator</param>
         /// <returns>The generated random seed</returns>
-        public uint GenerateRandomSeed(uint baseSeed = SamplerUtility.largePrime)
+        public uint NextRandomSeed()
```

Comment:
Mentioned earlier, but this method name should probably reflect what is happening in here better